connector/src/main/java/org/jboss/as/connector/subsystems/datasources/JdbcDriverAdd.java

```diff
@@ -97,14 +97,6 @@ protected void performRuntime(OperationContext context, ModelNode operation, Mod
         final ModuleIdentifier moduleId;
         final Module module;
         String slot = operation.hasDefined(MODULE_SLOT.getName()) ? MODULE_SLOT.resolveModelAttribute(context, operation).asString() : null;
```

Comment:
@bstansberry Thank you for the explanation, I will change the `operation` param to `model` in `performRuntime` method. :-)